core/cbor_blake/cid_block.hpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "cbor_blake/cid.hpp"
+
+namespace fc {
+  struct BlockParentCbCids : std::vector<CbCid> {
+    using vector::vector;
+    bool mainnet_genesis{};
+  };
+
+  constexpr BytesN<36> kMainnetGenesisBlockParent{
```

Comment:
I think it should be configurable in case the user will run network other than mainnet.

Reply:
Mainnet has such broken genesis, so this just works around it, and doesn't break normal blocks.